### PR TITLE
Add flag to override prompt for existing dir

### PR DESCRIPTION
**Pull request by Yaroslav-95**
Added --force flag to build subcommand. This restores use-cases that PR #1558
broke, without breaking current default behaviour.

**IMPORTANT: Please do not create a Pull Request adding a new feature without
discussing it first.**

The place to discuss new features is the forum: <https://zola.discourse.group/>
If you want to add a new feature, please open a thread there first in the
feature requests section.

Sanity check:

* [x] Have you checked to ensure there aren't other open 
  [Pull Requests](https://github.com/getzola/zola/pulls) for the same
  update/change?

## Code changes (Delete or ignore this section for documentation changes)

* [x] Are you doing the PR on the `next` branch?

If the change is a new feature or adding to/changing an existing one:

* [x] Have you created/updated the relevant documentation page(s)?

### CODE REVIEW

**Comment by Keats** [comment]
I think https://github.com/getzola/zola/pull/1985 covers your usecase as it won't delete the folder?

**Comment by Yaroslav-95** [comment]
On 22/09/12 10:07AM, Vincent Prouillet wrote:
> I think https://github.com/getzola/zola/pull/1985 covers your usecase as it won't delete the folder?
> 
Except it doesn't, because what PR #1558 introduced is a prompt for when
the output dir already exists, which is bad for non-interactive cases
(my use-case). PR #1985 doesn't cover this, since, AFAICT, it doesn't
override the prompt, which my PR does.

-- 

Yaroslav de la Peña Smirnov

File: src/cmd/build.rs [hunk 1]
```diff
@@ -13,13 +13,14 @@ pub fn build(
...
 ) -> Result<()> {
     let mut site = Site::new(root_dir, config_file)?;
     if let Some(output_dir) = output_dir {
         // Check whether output directory exists or not
         // This way we don't replace already existing files.
-        if output_dir.exists() {
+        if !force && output_dir.exists() {
             console::warn(&format!("The directory '{}' already exists. Building to this directory will delete files contained within this directory.", output_dir.display()));
 
             // Prompt the user to ask whether they want to continue.
```
Comment (Keats) [inline src/cmd/build.rs]: Let's remove the tokio timeout + question if there is a flag option though. It should just fail if the folder exists and --force is not passed.

**Review by Keats — CHANGES_REQUESTED** [review]

**Comment by Yaroslav-95** [comment]
On 22/09/14 04:12AM, Vincent Prouillet wrote:
> @Keats requested changes on this pull request.
> 
> Let's remove the tokio timeout + question if there is a flag option
> though. It should just fail if the folder exists and --force is not
> passed.
> 
Sure, I've committed the changes.

-- 

Yaroslav de la Peña Smirnov

**Review by Keats — APPROVED** [review]

**Comment by Keats** [comment]
Thanks!

**Pull request merged**